src/imp/tracer_imp.js

```diff
@@ -260,12 +283,10 @@ export default class TracerImp extends EventEmitter {
 
                 switch (suffix) {
                 case 'traceid':
-                    span.setFields({ trace_guid : value });
+                    spanContext._traceGUID = value;
```

Comment:
hm, reading this over I realize there's a bug when extraction fails (we need to return `null` in that case)... I will fix this later, have to go soon.